android-bindings/src/bindings.rs

```diff
@@ -1666,19 +1668,17 @@ pub unsafe extern "C" fn Java_com_mobilecoin_lib_TransactionBuilder_init_1jni(
     fog_resolver: JObject,
     memo_builder_box: JObject,
     block_version: jint,
+    token_id: jlong,
+    minimum_fee: jlong,
 ) {
     jni_ffi_call(&env, |env| {
         let fog_resolver: MutexGuard<FogResolver> =
             env.get_rust_field(fog_resolver, RUST_OBJ_FIELD)?;
         let block_version = BlockVersion::try_from(block_version as u32).unwrap();
-        // Note: RTHMemoBuilder can be selected here, but we will only actually
-        // write memos if block_version is large enough that memos are supported.
-        // If block version is < 2, then transaction builder will filter out memos.
         let memo_builder_box: Box<dyn MemoBuilder + Send + Sync> =
             env.take_rust_field(memo_builder_box, RUST_OBJ_FIELD)?;
-        // FIXME #1595: The token id should be a parameter and not hard coded to Mob
-        // here
-        let fee_amount = Amount::new(Mob::MINIMUM_FEE, Mob::ID);
+        let token_id = TokenId::try_from(token_id as u64).unwrap();
```

Comment:
i think this can just be `TokenId::from`, all u64's are potentially valid token ids

Reply:
Yeah, the problem is `jlong ~= i64`